### DISCUSSION TITLE
Resolving inconsistent returns using recursor and public DNS

### DISCUSSION THREAD

**Discussion by sysiocn** [PowerDNS/pdns#14894] [Q&A]
HI，Not familiar with recursor，Resolving inconsistent returns using recursor and public DNS。

pdns-recursor: powerdns/pdns-recursor-51:5.1.1

![image](https://github.com/user-attachments/assets/1ae19c81-fd99-4ad4-8c24-23579e41da16)

Close dnssec
![image](https://github.com/user-attachments/assets/c336ef0e-09eb-45b6-9236-3f5c373a7a72)

**Comment by omoerbeek** [comment]
https://dnsviz.net/d/mcs-bd.feishu.cn/dnssec/ shows quite a few problems with `mcs-bd.feishu.cn`, some of the issues (for example missing AA bits in replies and parent is NXDOMAIN while there are child domain) can cause Recursor to fail.
